.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+## Description:
+
+
+
+## PR Checklist
+
+- [ ] Have you verified the basic functionality for this change?
+
+- [ ] Added unit tests?
+
+- [ ] Added e2e tests
+
+- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
+
+### Screenshot/Video:
+
+
+### Reviewers:
+
+
+### Issue:
+
+- https://oktainc.atlassian.net/browse/OKTA-XXXXX
```

Comment:
this could be an external github issue as well right ?

Reply:
We usually create a JIRA for GH issues (that we are going to fix). But it would be nice to link to related GH issues if there are any.